README.md

```diff
@@ -1,5 +1,23 @@
-capistranorb.com
-====================
+# capistranorb.com
 
-Essay style documentation for Capistrano build with Jekyll and Github Pages.
-Send a PR here to contribute docs @ [capistranorb.com](http://capistranorb.com)
+This is the repository that generates the [capistranorb.com](http://capistranorb.com/) site. Feel free to send pull requests to make improvements to Capistrano's documentation!
+
+### Quick start
+
+This is a GitHub Pages project, which means it is built using Jekyll. To run the site locally, you'll need a functioning Ruby environment.
+
+After checking out the capistrano-documentation project, run:
+
+```
+bundle
```

Comment:
This may just be me, but I tend to write `bundle install` in documentation, because it is clearer, and just in case Bundler decides to change the default behavior.

Nitpick.